index.bs

```diff
@@ -1940,38 +1966,42 @@ To <dfn>parse summary buckets</dfn> given a [=map=] |map| and an integer |maxEve
 1. Return |summaryBuckets|.
 
 To <dfn>parse trigger specs</dfn> given a [=map=] |map|, a [=moment=]
-|sourceTime|, a [=duration=] |expiry|, a [=report window list=]
-|defaultReportWindows|, an unsigned 32-bit integer
-|defaultTriggerDataCardinality|, and a [=trigger-data matching mode=]
-|matchingMode|:
+|sourceTime|, a [=source type=] |sourceType|, a [=duration=] |expiry|, and a
+[=trigger-data matching mode=] |matchingMode|:
 
-1. [=Assert=]: |defaultTriggerDataCardinality| is greater than 0 and less than
-    or equal to [=max distinct trigger data per source=].
+1. Let |defaultReportWindows| be the result of
+    [=parsing top-level report windows=] with |map|, |sourceTime|, |sourceType|,
+    and |expiry|.
+1. If |defaultReportWindows| is an error, return an error.
 1. Let |specs| be a new [=map=].
 1. If |map|["`trigger_specs`"] does not [=map/exist=]:
     1. Let |spec| be a new [=trigger spec=] with the following items:
         : [=trigger spec/event-level report windows=]
         :: |defaultReportWindows|
     1. [=set/iterate|For each=] integer |triggerData| of [=the exclusive range|the range=] 0 to
-        |defaultTriggerDataCardinality|, exclusive:
+        [=default trigger data cardinality=][|sourceType|], exclusive:
         1. [=map/Set=] |specs|[|triggerData|] to |spec|.
     1. Return |specs|.
```

Comment:
Might make sense to add a comment to just state that using top-level report windows prevents use of full flex behavior.

Reply:
Not sure I'm following your comment here. Maybe it's for the wrong line?

Reply:
Why would using top-level windows prevent full-flex? I believe Android uses those as default for full-flex in case a trigger spec doesn't provide it.

Reply:
Sorry, this was just myself mis-reading the flow here, please ignore.